src/createPixelsRenderer.ts

```diff
@@ -38,9 +38,8 @@ export default function createPixelsRenderer(el: HTMLDivElement) {
       el.appendChild(maskDiv)
     }
     drawMask(w)
-    // TODO: `webkitMaskImage` is deprecated, maybe use `maskImage` instead
-    maskDiv.style.webkitMaskImage = `url("${maskCanvas.toDataURL()}")`
-    maskDiv.style.webkitMaskSize = `cover`
+    maskDiv.style.maskImage = `url("${maskCanvas.toDataURL()}")`
```

Comment:
.vscode/extensions/ms-vscode.vscode-typescript-next-4.7.20220427/node_modules/typescript/lib/lib.dom.d.ts

`HTMLDivElement` has `maskImage` property, but I dunno why it blame like these

Reply:
Update:
I find `maskImage` only in TS v4.7.
Now, 4.7 is a beta version.
So, we must wait for v4.7 to be released